linchpin/provision/roles/aws/tasks/teardown_aws_ec2_vpc_nat_gateway.yml

```diff
@@ -0,0 +1 @@
+provision_aws_ec2_vpc_nat_gateway.yml
```

Comment:
I don't see the logic here, should it not be something like this?
ec2_vpc_nat_gateway:
    state: absent
    nat_gateway_id: nat-12345678
    release_eip: yes
    wait: yes
    wait_timeout: 300
    region: ap-southeast-2

Reply:
this is how Github represents symbolic link. This links back to provision_aws_ec2_vpc_nat_gateway.yml and when linchpin destroy is called, it will trigger this part: https://github.com/CentOS-PaaS-SIG/linchpin/blob/a5c2b4619c0242147003c900fa0f6e33f6be1905/linchpin/provision/roles/aws/tasks/teardown_resource_group.yml#L144

The 'state' variable should be absent in that case: https://github.com/CentOS-PaaS-SIG/linchpin/blob/a5c2b4619c0242147003c900fa0f6e33f6be1905/linchpin/__init__.py#L924